### PR TITLE
feat: added Input and image component

### PR DESCRIPTION
**Pull request by AtulHande03a**
Image component contains:
responsive image
rounded ends image
circular image

Input component contains

inputs and textarea example
Screenshot (49)
Screenshot (50)
Screenshot (51)

### CODE REVIEW

File: components/images/image.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="description" content="Image Component" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
```
Comment (yashpatel1797) [inline components/images/image.html:9]: add the author meta tag.

File: documentation/docs.html [hunk 2]
```diff
@@ -741,6 +741,104 @@ <h4 class="horizontal-card-title font-regular mb-0-5">
...
+                  class="input my-1 mx-1"
+                  placeholder="Enter text here.."
+                  >
+                </textarea>
+            </div>
+            <div class="snippet my-1-5">
+                <iframe
+                  src="https://carbon.now.sh/embed/qNcOxb5EbH3C5bkKanPZ"
+                  style="height: 400px; border:0; transform: scale(1); overflow:hidden;               "
+                  sandbox="allow-scripts allow-same-origin">
```
Comment (yashpatel1797) [inline documentation/docs.html:834]: I know this is from carbon.sh but you can give class here and write CSS for that to avoid inline CSS.

**Review by yashpatel1797 — APPROVED** [review]
design and code looks good.

File: components/images/image.html (hunk 1 above)
Comment (AtulHande03a) [reply components/images/image.html:9]: yes I will do that

File: documentation/docs.html (hunk 2 above)
Comment (AtulHande03a) [reply documentation/docs.html:834]: I didn't wanted to override the inline style which carbon provides but I will see this later once I am done with submission.

File: documentation/docs.html [hunk 3]
```diff
@@ -741,6 +741,104 @@ <h4 class="horizontal-card-title font-regular mb-0-5">
...
+                </div>
+                <div class="flex-column mx-1 my-1">
+                  <label class="label" for="email-example">Password</label>
+                  <input
+                    class="input invalid"
+                    id="password-example"
+                    type="passwrod"
+                    placeholder="Input with invalid feedback"
+                  />
+                  <span class="invalid-feedback">Invalid passwrod</span>
```
Comment (zabihshaik) [inline documentation/docs.html]: Typo - password*

**Review by zabihshaik — COMMENTED** [review]
Everything looks good. Just a single comment. Please check.

**Review by horsemaker — APPROVED** [review]

**Pull request merged**